pages/_document.tsx

```diff
@@ -1,10 +1,31 @@
-import Document, { Html, Head, Main, NextScript } from "next/document";
+import Document, {
+  Html,
+  Head,
+  Main,
+  NextScript,
+  DocumentContext,
+} from "next/document";
+
+export default class MyDocument extends Document {
+  static async getInitialProps(ctx: DocumentContext) {
+    const initialProps = await Document.getInitialProps(ctx);
+    return { ...initialProps };
+  }
```

Comment:
```suggestion
```

I intentionally left this out because I have no intention of changing its behaviour.

<img width="693" alt="Screenshot 2021-07-08 at 00 37 06" src="https://user-images.githubusercontent.com/6315466/124797232-a0b7a880-df84-11eb-88aa-e65f22c420a1.png">